packages/manager/cypress/e2e/core/account/restricted-user-details-pages.spec.ts

```diff
@@ -0,0 +1,575 @@
+          'To upgrade your existing worker nodes, you can recycle all nodes (which may have a performance impact) or perform other upgrade methods.',
+          { exact: false }
+        ).should('be.visible');
+
+        ui.button
+          .findByTitle('Recycle All Nodes')
+          .should('be.visible')
+          .should('be.enabled')
+          .click();
+      });
+
+    // Verify clicking the "Recycle All Nodes" makes an API call
+    cy.wait('@recycleAllNodes');
+
+    // Verify the upgrade dialog closed
+    cy.findByText(stepTwoDialogTitle).should('not.exist');
+
+    // Verify the banner is still gone after the flow
+    cy.findByText(upgradePrompt).should('not.exist');
+
+    // Verify the version is correct after the update
+    cy.findByText(`Version ${newVersion}`);
+
+    ui.toast.findByMessage('Recycle started successfully.');
+  });
+
+  it("should disable action elements and buttons in the 'Longview' details page", () => {
+    const longviewLastUpdatedWaiting = longviewResponseFactory.build({
+      ACTION: 'lastUpdated',
+      DATA: { updated: 0 },
+      NOTIFICATIONS: [],
+      VERSION: 0.4,
+    });
+
+    const longviewGetValuesWaiting = longviewResponseFactory.build({
+      ACTION: 'getValues',
+      DATA: {},
+      NOTIFICATIONS: [],
+      VERSION: 0.4,
+    });
+
+    const longviewGetLatestValueWaiting = longviewResponseFactory.build({
+      ACTION: 'getLatestValue',
+      DATA: {},
+      NOTIFICATIONS: [],
+      VERSION: 0.4,
+    });
+
+    const client: LongviewClient = longviewClientFactory.build({
+      api_key: '01AE82DD-6F99-44F6-95781512B64FFBC3',
+      apps: longviewAppsFactory.build(),
+      created: new Date().toISOString(),
+      id: 338283,
+      install_code: '748632FC-E92B-491F-A29D44019039017C',
+      label: 'longview-client-longview338283',
+      updated: new Date().toISOString(),
+    });
+
+    mockGetLongviewClients([client]).as('getLongviewClients');
+    mockFetchLongviewStatus(client, 'lastUpdated', longviewLastUpdatedWaiting);
+    mockFetchLongviewStatus(client, 'getValues', longviewGetValuesWaiting);
+    mockFetchLongviewStatus(
+      client,
+      'getLatestValue',
+      longviewGetLatestValueWaiting
+    ).as('fetchLongview');
+
+    cy.visitWithLogin('/longview');
+    cy.wait(['@getProfile', '@getLongviewClients']);
+
+    // Confirm that the warning message is displayed
+    cy.findByText(
+      `You don't have permissions to create Longview Clients. Please contact your ${ADMINISTRATOR} to request the necessary permissions.`
+    );
+
+    // Confirm that the "Add Client" button is disabled
+    ui.button
+      .findByTitle('Add Client')
+      .should('be.visible')
+      .should('be.disabled')
+      .trigger('mouseover');
+    ui.tooltip.findByText(
+      `You don't have permissions to create Longview Clients. Please contact your ${ADMINISTRATOR} to request the necessary permissions.`
+    );
+
+    // TODO: M3-9586 The action buttion is not diabled
```

Comment:
same here, also looks like this ticket was completed

```suggestion
    // TODO: M3-9586 The action buttion is not disabled
```

Reply:
Done